src/client/components/scenario-overview/scenario-overview.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <aside class="scenario-overview">
+    <md-button @click="showPopup = true">
+      {{ $t('scenario_examples') }}
+    </md-button>
+    <app-popup
+      v-if="showPopup"
+      :title="$t('scenario_examples')"
+      @closePopup="showPopup = false"
+    >
+      <div class="scenario-overview__content">
+        <md-tabs :md-active-tab="initialTab" @md-changed="newActiveTab => activeTab = newActiveTab">
+          <md-tab
+            v-for="scenario in scenarios"
+            :id="scenario.value"
+            :key="scenario.value"
+            :md-label="scenario.title"
```

Comment:
There's quite some scenarios, with long titles. Any chance of giving the `md-tab` some padding left and right, and truncating the text?

Reply:
Done!